app/views/playlist-view.js

```diff
@@ -20,7 +21,10 @@ const PlaylistCardView = Backbone.View.extend({
       collection: test,
     });
     songlist.render();
-    Utils.slidePlayListIn(document.getElementById('playlists'));
+    setTimeout(() => {
+      Utils.slidePlayListIn(document.getElementById('playlists'));
+    }, 1000);
```

Comment:
asta nu e o certitudine: ai o şansă să ajungi la buguri greu de reprodus din cauză unor probleme de sincronizări

ideal ai folosi animaţii de css şi ai asculta după evenimentul de animationEnd
mai puţin ideal, ai folosi animaţie de jQuery şi ai da un callback pentru când e gata

timeoutul acesta ar trebui refactorizat